okhttp/src/commonMain/kotlin/okhttp3/internal/-HostnamesCommon.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 The Android Open Source Project
+ * Copyright (C) 2021 Square, Inc.
```

Comment:
The GitHub diff suggests a copyright owner change. All of the code in this file is authored by Square.